docs/notebooks/low_level.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>grammar nit: "[The] Value of each key..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-pandas/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>